packages/react-scripts/template/index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module '*.jpg'
+
+declare module '*.png'
```

Comment:
```
declare module '*.png' {
  const src: string
  export default src
}
```
Is more accurate here as webpack file-loader will export the string path of the png file